integ-tests/src/main/java/com/google/testdata/TestDataHelper.java

```diff
@@ -28,20 +31,51 @@ private TestDataHelper() {}
    * @throws SQLException
    */
   public static void generateUsers(Connection connection, int userCount) throws SQLException {
-    final String sqlDataPath = "src/main/java/com/google/sql/data/users_data.sql";
-    final String password = UUID.randomUUID().toString();
+    final String usersDataPath = "src/main/java/com/google/sql/data/users_data.sql";
+    final String userData = getSql(usersDataPath);
+    final String password = randomUUID().toString();
 
-    ImmutableList<String> sqlQueries =
-        Stream.generate(memoize(() -> SqlHelper.getSql(sqlDataPath)))
+    ImmutableList<String> usersQueries =
+        Stream.generate(memoize(() -> userData))
             .limit(userCount)
             .map(sqlQuery -> format(sqlQuery, nanoTime(), password))
             .collect(toImmutableList());
 
-    SqlHelper.executeQueries(connection, sqlQueries);
+    executeQueries(connection, usersQueries);
 
     logger.info(
         format(
-            "Generated %s new Users:\n%s",
-            sqlQueries.size(), Joiner.on("\n").join(sqlQueries).replaceAll(password, "####")));
+            "Generated %s new User(s):\n%s",
+            usersQueries.size(), Joiner.on("\n").join(usersQueries).replaceAll(password, "####")));
+  }
+
+  /**
+   * @param connection DB connection parameter
+   * @param dbTablePairsCount Repetition count
+   * @throws SQLException
+   */
+  public static void generateDbTablePairs(Connection connection, int dbTablePairsCount)
+      throws SQLException {
+    final String columnsDataPath1 = "src/main/java/com/google/sql/data/columns_data_1.sql";
+    final String columnsDataPath2 = "src/main/java/com/google/sql/data/columns_data_2.sql";
+    final String dbData = getSql(columnsDataPath1);
+    final String tableData = getSql(columnsDataPath2);
+
+    List<String> dbTableQueries = new ArrayList<>();
+    while (dbTablePairsCount > 0) {
+      String dbName = "test_" + nanoTime();
+      String tableName = "test_" + nanoTime();
+
+      dbTableQueries.add(format(dbData, dbName));
+      dbTableQueries.add(format(tableData, dbName, tableName));
+      dbTablePairsCount--;
+    }
```

Comment:
That solution doesn't implement ImmutableList, I need to format 2 strings on the go and keep the value of the first string for formatting the second one. If there is a more elegant solution, I don't mind implementing it.

Reply:
Noted.
Will practice with this structure.